gcloud.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency      "minitest-rg", "~> 5.2"
   gem.add_development_dependency      "autotest-suffix", "~> 1.1"
   gem.add_development_dependency      "rdoc", "~> 4.0"
-  gem.add_development_dependency      "rubocop", "~> 0.27"
+  gem.add_development_dependency      "rubocop", "<= 0.35"
```

Comment:
I think we want to make it 0.35, but not any higher. I'd rather configure it as:

```ruby
gem.add_development_dependency      "rubocop", "~> 0.35.0"
```